serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/service/EntityService.java

```diff
@@ -58,7 +64,11 @@ public EntityService(BrregService brregService) {
      * @return info needed to send messages to the entity
      */
     public Optional<EntityInfo> getEntityInfo(String identifier) {
-        return this.entityCache.getUnchecked(identifier);
+        try {
+            return entityCache.get(identifier);
+        } catch (ExecutionException e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)